package.json

```diff
@@ -16,16 +16,21 @@
   },
   "homepage": "https://skia.org",
   "esy": {
+    "buildEnv": {
+      "PATH": "#{self.target_dir : $PATH}"
+    },
     "buildsInSource": true,
     "build": [
+      "cp /usr/bin/python ./python.exe",
```

Comment:
With this command here, we will create this copy on all platforms, even if it's not required on Mac OS and Linux, right? Do you think it would make sense to make this platform-dependent?

Reply:
You are right. I'll make it Windows only.